decidim-core/spec/mailers/reported_mailer_spec.rb

```diff
@@ -7,20 +7,64 @@ module Decidim
     let(:organization) { create(:organization) }
     let(:user) { create(:user, :admin, organization: organization) }
     let(:component) { create(:component, organization: organization) }
-    let(:reportable) { create(:dummy_resource, component: component) }
+    let(:reportable) { create(:proposal, title: Decidim::Faker::Localized.sentence, body: Decidim::Faker::Localized.paragraph(3)) }
```

Comment:
Hi @verarojman  `decidim-core` should not depend upon other modules, even for testing. It would be best to keep using the `dummy_resource` and improve it if required.

Reply:
Yes, you're right! I'll change this :)

Reply:
I need the dummy_resource to have a `creator_identity` attribute to test this, could you help me understand how I need to set it up? Thanks!

Reply:
Ok, then, let's keep using proposal for the moment